metricbeat/docs/fields.asciidoc

```diff
@@ -6040,6 +6040,194 @@ Used inodes
 
 
 [float]
+== events Fields
+
+The Kubernetes events metricset collects events that are generated by objects running inside of Kubernetes
+
+
+
+[float]
+=== kubernetes.events.count
+
+type: long
+
+Count field records the number of times the particular event has occured
+
+
+[float]
+=== kubernetes.events.firstOccurrenceTimestamp
+
+type: date
+
+Timestamp of first occurrence of event
+
+
+[float]
+=== kubernetes.events.lastOccurrenceTimestamp
+
+type: date
+
+Timestamp of last occurrence of event
+
+
+[float]
+=== kubernetes.events.message
+
+type: keyword
+
+Message recorded for the given event
+
+
+[float]
+=== kubernetes.events.reason
+
+type: keyword
+
+Reason recorded for the given event
+
+
+[float]
+=== kubernetes.events.type
+
+type: keyword
+
+Type of the given event
+
+
+[float]
+== metadata Fields
+
+Metadata associated with the given event
+
+
+
+[float]
+=== kubernetes.events.metadata.creationTimestamp
+
+type: date
+
+Timestamp of creation of the given event
+
+
+[float]
+=== kubernetes.events.metadata.deletionTimestamp
+
+type: date
+
+Timestamp of deletion of the given event
+
+
+[float]
+=== kubernetes.events.metadata.name
+
+type: keyword
+
+Name of the event
+
+
+[float]
+=== kubernetes.events.metadata.namespace
+
+type: keyword
+
+Namespace in which event was generated
+
+
+[float]
+=== kubernetes.events.metadata.resourceVersion
+
+type: keyword
+
+Version of the event resource
+
+
+[float]
+=== kubernetes.events.metadata.uid
+
+type: keyword
+
+Unique identifier to the event object
+
+
+[float]
+=== kubernetes.events.metadata.selfLink
+
+type: keyword
+
+URL representing the event
+
+
+[float]
+== involvedObject Fields
+
+Metadata associated with the given involved object
+
+
+
+[float]
+=== kubernetes.events.involvedObject.apiVersion
+
+type: keyword
+
+API version of the object
+
+
+[float]
+=== kubernetes.events.involvedObject.kind
+
+type: keyword
+
+API kind of the object
+
+
+[float]
+=== kubernetes.events.involvedObject.name
+
+type: keyword
+
+name of the object
+
+
+[float]
+=== kubernetes.events.involvedObject.resourceVersion
+
+type: keyword
+
+resource version of the object
+
+
+[float]
+=== kubernetes.events.involvedObject.uid
+
+type: keyword
+
+UUID version of the object
+
+
+[float]
+== tags Fields
+
+Tags to help select the event based on pod name and host name
+
+
+
+[float]
+=== kubernetes.events.tags.host
+
+type: keyword
+
+Host on which the event was generated
+
+
+[float]
+=== kubernetes.events.tags.pod
```

Comment:
these are dimensions that can be used to better query information.  base implementation off of which I went with is here:
https://github.com/kubernetes/heapster/blob/master/events/sinks/elasticsearch/driver.go#L76